packages/integrations/node/src/preview-entrypoint.ts

```diff
@@ -0,0 +1,44 @@
+import { fileURLToPath } from 'node:url';
+import { AstroError } from 'astro/errors';
+import logListeningOn from './log-listening-on.js';
+import { createServer } from './standalone.js';
+import type { CreatePreviewServer } from 'astro';
+import type { createExports } from './server-entrypoint.js';
+
+type ServerModule = ReturnType<typeof createExports>;
+type MaybeServerModule = Partial<ServerModule>;
+
+const preview1: CreatePreviewServer = async function (preview) {
```

Comment:
Why not `createPreviewServer`?

```suggestion
const createPreviewServer: CreatePreviewServer = async function (preview) {
```